deltakit-explorer/tests/unit/circuits/test_tableau_functions.py

```diff
@@ -8991,6 +9000,14 @@ def test_cpswap_to_czswap_dict_compilations_give_equivalent_tableaus(self, gate)
             == czswap_with_unitaries_tableau
         )
 
+    @pytest.mark.skipif(
+        semver.Version.parse(stim.__version__) < semver.Version(1, 13, 0),
+        reason=(
+            "CZSWAP gate has been introduced in Stim v1.13.0."
+            "See https://github.com/quantumlib/Stim/releases/tag/v1.13.0."
+            f"Current Stim version is {stim.__version__}."
+        )
```

Comment:
Ideally, this is factored in a re-usable marker somewhere, and re-used across the code base.